### PR TITLE
adds 404

### LINKED ISSUE #18: Create error view for 404'd snippets

### PR DESCRIPTION
**Pull request by dane-johnson**
screen shot 2017-04-26 at 11 16 07 am
Closes #18

### CODE REVIEW

**Comment by Hopding** [comment]
Seems a bit "in your face". I think it would look better if the hotdog were outlined, rather than filled in.

**Comment by solkaz** [comment]
🌭

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <script src='codesplain.js'></script>
   <body>
     <div data-component="codesplain" data-snippet="maryvilledev/test-1"></div>
-    <div data-component="codesplain" data-snippet="maryvilledev/say_carrot"
+    <div data-component="codesplain" data-snippet="bullshitorg/bullshitsnippet"></div>
```
Comment (solkaz) [inline public/index.html]: Think we can make this more SFW?

File: src/containers/AppBody.jsx [hunk 2]
```diff
@@ -26,16 +31,25 @@ class AppBody extends Component {
...
+      return (
+        <div style={styles.container}>
+          <RulesSelector />
+          <SnippetArea />
+          <AnnotationDisplay />
+        </div>
+      );
+    } else {
+      return <Error snippetKey={snippetKey} />
+    }
```
Comment (solkaz) [inline src/containers/AppBody.jsx]: You can remove the else clause since you're returning in the if statement

**Comment by dane-johnson** [comment]
@Hopding You just want me to have to spend more time doing CSS.

**Comment by dane-johnson** [comment]
screen shot 2017-04-27 at 9 54 58 am

**Review by Hopding — APPROVED** [review]
🚢 ⚓️ ⛵️ 🛥 🌊

**Review by solkaz — APPROVED** [review]

**Pull request merged**